code/modules/clothing/head/jobs.dm

```diff
@@ -19,26 +19,28 @@
 	return(FIRELOSS)
 
 //Captain
-/obj/item/clothing/head/caphat
+/obj/item/clothing/head/soft/caphat
 	name = "captain's hat"
 	desc = "It's good being the king."
-	icon_state = "captain"
+	icon_state = "captainsoft"
+	item_color = "captian"

```

Comment:
nope look at softcaps shitty code
color is how it updates item sprites insted of doing something sane like having a flipped sprite var and reverting using inital